src/instrumentsscene/view/abstractinstrumentspaneltreeitem.cpp

```diff
@@ -258,6 +258,12 @@ void AbstractInstrumentsPanelTreeItem::setIsVisible(bool isVisible)
     }
 }
 
+void AbstractInstrumentsPanelTreeItem::setIsVisibleFromStaffTreeItem(bool isVisible)
+{
+    m_isVisible = isVisible;
```

Comment:
As requested by a user on Discord, better add:
```
    if (m_isVisible == isVisible) {
        return;
    }
```
to avoid unneeded signaling

Reply:
Added to the function